aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -116,24 +116,105 @@ Rendering server components from a static HTML page isn't supported.
-
-```html
-<script src="_framework/blazor.server.js" autostart="false"></script>
-<script>
-  Blazor.start({
-    configureSignalR: function (builder) {
-      builder.configureLogging("information"); // LogLevel.Information
-    }
-  });
-</script>
+* Pass in a configuration object (`configureSignalR`) that calls `configureLogging` with the log level on the client builder.
+
+```cshtml
+    ...
+
+    <script src="_framework/blazor.server.js" autostart="false"></script>
+    <script>
+      Blazor.start({
+        configureSignalR: function (builder) {
+          builder.configureLogging("information");
+        }
+      });
+    </script>
+</body>
 ```
 
+In the preceding example, `information` is equilivant to a log level of <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>.
+
+### Modify the reconnection handler
+
+To modify the reconnection handler's circuit connection events:
+
+* Add an `autostart="false"` attribute to the `<script>` tag for the `blazor.server.js` script.
+* Register callbacks for connection changes for either or both of the following events:
+  * Dropped connection (`onConnectionDown`)
+  * Established/re-established connection (`onConnectionUp`)
+
+```cshtml
+    ...
+
+    <script src="_framework/blazor.server.js" autostart="false"></script>
+    <script>
+      Blazor.start({
+        reconnectionHandler: {
+          onConnectionDown: (options, error) => console.error(error);
+          onConnectionUp: () => console.log("Up, up, and away!");
+        }
+      });
+    </script>
+</body>
+```
+
+### Adjust the reconnection retry count and interval
+
+To adjust the reconnection retry count and interval:
+
+* Add an `autostart="false"` attribute to the `<script>` tag for the `blazor.server.js` script.
+* Set the number of retries (`maxRetries`) and period in milliseconds permitted for each retry attempt (`retryIntervalMilliseconds`).
+
+```cshtml
+    ...
+
+    <script src="_framework/blazor.server.js" autostart="false"></script>
+    <script>
+      Blazor.start({
+        reconnectionOptions: {
+          maxRetries: 3,
+          retryIntervalMilliseconds: 2000
+        }
+      });
+    </script>
+</body>
+```
+
+### Hide or replace the reconnection display
+
+To hide the reconnection display:
+
+* Add an `autostart="false"` attribute to the `<script>` tag for the `blazor.server.js` script.
+* Set the reconnection handler's `_reconnectionDisplay` to an empty object (`{}` or `new Object()`).
+
+```cshtml
+    ...
+
+    <script src="_framework/blazor.server.js" autostart="false"></script>
+    <script>
+      window.addEventListener('beforeunload', function () {
+        Blazor.defaultReconnectionHandler._reconnectionDisplay = {};
+      });
+    </script>
+</body>
+```
+
+To replace the reconnection display, set `_reconnectionDisplay` in the preceding example to the element for display:
+
+```javascrpt
```

Comment:
```suggestion
```javascript
```

Reply:
Pro tip: If you have the VS Code Docs Authoring Pack extension installed, typing 3 backticks in a row displays a list of valid dev langs. By using that strategy, you'll never end up with an invalid dev lang.